### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by Elland**

### CODE REVIEW

**Comment by 3lvis** [comment]
There's already a template for this. https://github.com/3lvis/swift-library-template/blob/master/LICENSE.md

**Comment by Elland** [comment]
Are they different?

**Comment by 3lvis** [comment]
Similar, but different in formatting. Just copy the existing one to keep consistency.

**Comment by Elland** [comment]
🙆

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+> Copyright (c) 2016 <AUTHOR_NAME>
```
Comment (3lvis) [inline LICENSE]: Can you change the AUTHOR_NAME

Comment (Elland) [inline LICENSE]: To what exactly? 🤔

Comment (3lvis) [inline LICENSE]: Bakken & Bæck AS

**Comment by Elland** [comment]
🙆 🆗

**Comment by 3lvis** [comment]
Looks good, merge it!

**Pull request merged**